e2e/go.mod

```diff
@@ -12,20 +12,20 @@ require (
 	github.com/cosmos/ibc-go/modules/light-clients/08-wasm v0.0.0-00010101000000-000000000000
 	github.com/cosmos/ibc-go/v8 v8.0.0
 	github.com/docker/docker v24.0.7+incompatible
-	github.com/strangelove-ventures/interchaintest/v8 v8.0.0
+	github.com/strangelove-ventures/interchaintest/v8 v8.0.1-0.20231206154526-2f014d308bea
```

Comment:
I needed to bump interchaintest to get this [PR](https://github.com/strangelove-ventures/interchaintest/pull/883) that added support for a new field (chain `type`) that has been added to hermes config.toml.